lib/workers/pr/changelog/source-github.js

```diff
@@ -42,12 +42,13 @@ async function getRepositoryHead(repository, version) {
   if (version.gitHead) {
     return version.gitHead;
   }
-  if (!version.date) {
+  const date = version.date || version.time;
```

Comment:
Is this caused by inconsistency in our datasource code? Should we standardise this per-datasource now or later?